environments-networks/hmpps-test.json

```diff
@@ -147,6 +147,17 @@
       "from_port": "9100",
       "to_port": "9100"
     },
+    {
+      "subnet_set": "general",
+      "egress": false,
+      "subnet_type": "private",
+      "protocol": "tcp",
+      "rule_action": "allow",
+      "rule_number": 250,
```

Comment:
This rule number already exists so the plan is failing

Reply:
done